.travis/travis-tasks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#Exit on failures
+set -e
+
+pushd /builddir/
+
+# We have to define the _Float* types as those are not defined by coverity and as result
+# the codes linking agains those (pretty much anything linking against stdlib.h and math.h)
+# won't be covered.
+echo "#define _Float128 long double" > /tmp/coverity.h
+echo "#define _Float64x long double" >> /tmp/coverity.h
+echo "#define _Float64 double" >> /tmp/coverity.h
+echo "#define _Float32x double" >> /tmp/coverity.h
+echo "#define _Float32 float" >> /tmp/coverity.h
+
+# The coverity scan script returns an error despite succeeding...
+ CFLAGS="${CFLAGS:- -include /tmp/coverity.h}" \
+ TRAVIS_BRANCH="${TRAVIS_BRANCH:-master}" \
+ COVERITY_SCAN_PROJECT_NAME="${COVERITY_SCAN_PROJECT_NAME:-SSSD/sssd}" \
+ COVERITY_SCAN_NOTIFICATION_EMAIL="${COVERITY_SCAN_NOTIFICATION_EMAIL:-sssd-maint@redhat.com}" \
```

Comment:
Do you know if the notification includes details someone could use to reproduce the issue? Is it maybe the same notification we receive from Stephen's coverity instance?

I'm just thinking if we want to limit the access to sssd-maint@redhat.com or create even a new address so that no info can be leaked to outside the code sssd team.

Reply:
As far as I understand the report we get is the one that's on coverity webpage. So, it'd be pretty much same one we receive from Stephen's coverity instance.

Maybe create a new one? Just with SSSD developers?

Reply:
Actually, dismiss this. We should be careful with reviewing the currently opened issues and think if any of them can be a CVE, but since any new issues would hopefully be triaged and found between pushing the patch and releasing a new version, then I think using sssd-maint is fine.

Reply:
For the record, the email that gets sent just gives a link to the results on scan.coverity.com plus a delta of fixed/new issues detected. It provides no details on the failures.

Admins can set permissions on scan.coverity.com whether the public should be able to view a) nothing, b) a summary of the LoC vs detected/fixed issues or c) all of the details. I set it up originally to be b)